cnf-certification-test/accesscontrol/suite.go

```diff
@@ -388,15 +388,19 @@ func TestPodClusterRoleBindings(env *provider.TestEnvironment) {
 	}
 }
 
-//nolint:funlen
-func TestAutomountServiceToken(env *provider.TestEnvironment) {
-	ginkgo.By("Should have automountServiceAccountToken set to false")
+//nolint:funlen,gocyclo
+func TestAutomountServiceToken(env *provider.TestEnvironment, testerFuncs rbac.AutomountTokenFuncs) {
+	if !tnf.IsUnitTest() {
+		ginkgo.By("Should have automountServiceAccountToken set to false")
+	}
 
 	msg := []string{}
 	failedPods := []string{}
 	for _, put := range env.Pods {
-		ginkgo.By(fmt.Sprintf("check the existence of pod service account %s (ns= %s )", put.Namespace, put.Name))
-		gomega.Expect(put.Spec.ServiceAccountName).ToNot(gomega.BeEmpty())
+		if !tnf.IsUnitTest() {
```

Comment:
Why not create a GinkgoBy() and GinkgoExpect function that would make the ginkgo calls and the if

Reply:
Done.